doc/Security.xml

```diff
@@ -947,6 +979,55 @@ Added certificate-based client authentication</revremark>
         </orderedlist>
       </section>
     </section>
+    <section xml:id="_Ref395181339">
+      <title>JWT-based client authorization</title>
+      <para>Unlike with HTTP Digest and WS-UsernameToken authentication, this specification defines how to associate clients with the different User Levels as defined in the ONVIF Core Specification. Devices are not expected to have the user credentials stored in their memory, instead they will verify that the user has been correctly authenticated by an external service based on Open ID Connect and authorize accessing different functions, based on the user level presented in  the supplied JWT embedded in the binary security token.</para>
+      <para>The content of the binary security token is the base64-encoded JWT according to [RFC 7519].</para>
+      <para>The JOSE header shall declare that the encoded object is a JWT, and the JWT is a JWS that is signed using ECDSA using P-256 (secp256r1) curve and SHA-256 hash algorithm. Additionally, it must include the <emphasis>kid</emphasis> claim as defined by [RFC 7515], which indicates the ID associated to the key used to sign the JWT. If the absolute path of the JWK Set URL is different from https://iss/.well-known/jwks.json, the header may include an optional <emphasis>jku</emphasis> claim.</para>
```

Comment:
I think we should not mandate the use of ECDSA as this will drastically limit the number of identity providers that we can use. I think we can have it as a recommendation but we also need to support RS256.